e2e/manual-tests/catalogue/catalogue-test_4.md

```diff
@@ -22,14 +22,17 @@ A researcher working in consortium X (*example: testNetwork1*) searches for spec
 | 5 | Click on the testNetwork1 name | Should be redirected to the testNetwork1 'Welcome to... Select one of the content categories listed below.'. Cohorts should show 4, variables should show 3. | | |
 | 6 | Hover over yellow Variables button | Button should be highlighted | | |
 | 7 | Click Variables button | Should be redirected to testNetwork1 variable explorer; Variable page should show complete variable names, variable labels and a filters panel on the left hand side of the page. | | |
-| 8 | Search 'ADHD' in search bar | Variables should be filtered on ADHD variables ('testVarRepeats' and 'testVarNoRepeats'); topics should be filtered on ADHD; active filters should show 'ADHD'  | | |
-| 9 | Open topic 'Symptoms and signs' > 'Symptoms and signs involving cognition, perception, emotional state and behaviour (R40-R46)' > 'Behavioral problem domains' > select 'ADHD symptoms' | Variables should be filtered on ADHD variables ('testVarRepeats_' and 'testVarNoRepeats'); active filters should show topic name | | |
-| 10 | Click on the name of first variable 'testVarRepeats_' | Should redirect to variable detailed view; Details should show: 1) description, 2) harmonization status per cohort, 3) harmonization details per cohort. Table in 2: testCohort1 should show all repeats completely harmonized; testCohort2 should show all repeats NA; testCohort3 and testCohort4 should show partial, NA and complete. Example: testCohort3 should show repeat 1 completely, repeat 3 NA and repeat 4 partial. 4) tabs for each cohorts with further details (variables used, syntax) | | |
-| 11 | Hover 'about statuses' | Pop up should show 'completed: cohort was able to fully map to the harmonized variables'; 'partially: cohort was able to partially map to the harmonized variable'; 'No data: no harmonization data is available'. | | |
+| 8 | Search 'ADHD' in search bar | Variables should be filtered on ADHD variables ('testVarRepeats' and 'testVarNoRepeats'); active filters should show 'ADHD'  | | |
+| 8a| Remove 'ADHD' from the search bar | The list of variables returns to its initial form and no filters are shown above the list | | |
+| 9 | Click on 'Search for options' above the topics on the left hand side | A pop-up with the list of topics is displayed | | |
+| 9a| Type 'Symptoms' in the search field in the pop-up | Four sections of the topic tree with checkboxes are displayed | | |
+| 9b| Open topic 'Symptoms and signs' > 'Symptoms and signs involving cognition, perception, emotional state and behaviour (R40-R46)' > 'Behavioral problem domains' > select 'ADHD symptoms' and then choose Show Results| Variables should be filtered on ADHD variables ('testVarRepeats_' and 'testVarNoRepeats'); active filters should show -1 | | |
```

Comment:
As a general point, I think it would be more helpful if these variable were renamed "test_adhd_variable" etc, else it is always confusing why the names don't correspond to the search